scripts/sizeSnapshot/webpack.config.js

```diff
@@ -94,7 +94,7 @@ async function getSizeLimitBundles() {
       // vs https://bundlephobia.com/result?p=react-autosize-textarea
       name: '@material-ui/core/Textarea',
       webpack: true,
-      path: 'packages/material-ui/build/esm/InputBase/Textarea.js',
+      path: 'packages/material-ui/build/esm/TextareaAutosize/index.js',
```

Comment:
Will this affect size history? @eps1lon

Reply:
It doesn't the diff proves it :).

Reply:
Ah, I was trying to remember what the problem was before. :+1: